Source/ActivityIndicatorView.swift

```diff
@@ -281,16 +281,21 @@ public struct ActivityIndicatorView: View {
     }
 
     func createGradientIndicator(colors: [Color]) -> AnyView {
```

Comment:
I think it would be better to add rounded line cap as parameter, rather than using it all the time. Moreover, the default behaviour is better left untouched, so that nothing changes for those who are already content with unrounded indicator.

Reply:
I’ve added a lineCap parameter with default value = .butt